mdtogo/cmddocs/cmddocs_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmddocs_test
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"sort"
+	"testing"
+
+	"github.com/GoogleContainerTools/kpt/mdtogo/cmddocs"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParsingDocWithNameFromFolder(t *testing.T) {
```

Comment:
An existing issue that this PR should address is #1402. In that scenario two files in the same directory use the same markdown name. It may make sense to cover that in a test case here using two different prefixes/files.

```
<!--mdtogo:firstShort-->
<!--mdtogo:secondShort-->
```

Reply:
Added a test case for each of these!